app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Adapters/RouteSwitcherAdapter.java

```diff
@@ -0,0 +1,42 @@
+package ithaca_transit.android.cornellappdev.com.ithaca_transit.Adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.Models.Place;
+import ithaca_transit.android.cornellappdev.com.ithaca_transit.R;
```

Comment:
should this have been imported?

Reply:
All imports added have utility, removing them breaks the code.

Reply:
i'm not convinced that R should ever be imported...it doesn't seem like anything in this class uses R directly